pkg/skaffold/test/structure/structure.go

```diff
@@ -32,6 +32,15 @@ import (
 func (tr *Runner) Test(ctx context.Context, out io.Writer, image string) error {
 	logrus.Infof("Running structure tests for files %v", tr.testFiles)
 
+	if !tr.imagesAreLocal {
+		// First we have to pull the image.
+		// `container-structure-test` should be able to do it by itself
```

Comment:
would you mind opening an issue on that repo to track? this tool is seeing a resurgence of use recently, so I want to make sure we're capturing the most common frictions people are seeing in case anyone has motivation to work on it :)

Reply:
https://github.com/GoogleContainerTools/container-structure-test/issu